components/__init__.py

```diff
@@ -247,6 +244,16 @@ def define_components(reqs):
                 commands=isal_build,
                 required_progs=['nasm', 'yasm'],
                 libs=["isal"])
+    reqs.define('isal_crypto',
+                retriever=GitRepoRetriever(
+                        "https://github.com/intel/isa-l_crypto"),
```

Comment:
(pylint-bad-continuation) Wrong hanging indentation (remove 4 spaces).

Reply:
It seems to be okay with strings split across multiple lines, so I'm going to try that route. So ...
```
retriever=GitRepoRetriever("https://github.com/intel/"
                           "isa-l_crypto"),
```